modules/misc/xdg.nix

```diff
@@ -91,6 +93,9 @@ in
 
     {
       home.file = mkMerge [ cfg.configFile cfg.dataFile ];
+      home.activation.xdgCreateCache = dag.entryAfter [ "linkGeneration" ] ''
```

Comment:
I'm not certain about doing this unconditionally. Maybe with a configuration flag?

In any case, I guess the directory should be created with 700 umask and perhaps a [CACHEDIR.TAG](http://www.brynosaurus.com/cachedir/spec.html) file inside?

Reply:
The specs https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html says to create with 700 indeed.
As for the cachedir.tag, the website blocks me but it looks interesting.